mkdocs.yml

```
@@ -0,0 +1,65 @@
+site_name: cql2-rs
+site_url: https://developmentseed.org/cql2-rs/
+site_description: Command-line interface (CLI), Python package, and Rust crate for parsing, validating, and converting Common Query Language (CQL2)
+site_author: David Bitner
+repo_name: developmentseed/cql2-rs
+repo_url: https://github.com/developmentseed/cql2-rs
+edit_uri: edit/main/docs/
+
+extra:
+  social:
+    - icon: "fontawesome/brands/github"
+  version:
+    provider: mike
+
+theme:
+  name: material
+  logo: ds-logo-hor--pos.svg
+  favicon: favicon.png
+  icon:
+    repo: fontawesome/brands/github
+  palette:
+    primary: blue
+  features:
+    - content.action.edit
+    - navigation.indexes
+    - navigation.instant
+    - navigation.tabs
+    - navigation.tracking
+    - search.share
+    - search.suggest
+    - toc.integrate
+nav:
+  - cql2-rs: index.md
+  - Python: python.md
+  - CLI: cli.md
+  - Rust: rust.md
```

```suggestion
  - Rust: https://docs.rs/cql2
```

you can directly link the docs api in the menu 

but what you've done is also fine 👍

---

I like yours better, fixed.